ArmRobot/Assets/Scripts/Light/DirectionalLightRandomization.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DirectionalLightRandomization : MonoBehaviour
```

Comment:
I don't see anything about randomizing the position of the directional light. It would be good if we could randomize the directional light's position (so that the shadows can have major changes of direction) while still rotating the light so that it is roughly pointed at the table/cube/robot.

Reply:
It is done in the PointLightRandomization. In the DomainRandomization script I generate PointLights and then I move their position. Change the position of the directional light has no effect on the luminosity of the scene

Reply:
Oh, ok! Good to know.